Library/Formula/offzip.rb

```diff
@@ -0,0 +1,15 @@
+class Offzip < Formula
+  version "0.3.6a"
+  homepage "http://aluigi.altervista.org/mytoolz.htm#offzip"
+  url "http://aluigi.altervista.org/mytoolz/offzip.zip"
+  sha256 "71d578d4bcfad52857ef7d2db0b16935ad88aaef493c9b4367d310069efd892b"
+
+  def install
+    system ENV.cc, "offzip.c", "-o", "offzip", "-lz"
+    bin.install "offzip"
+  end
+
+  test do
+    system "offzip|grep #{version}"
```

Comment:
The url contains no version number, so the test is just to ensure the right version is used.
I guess I should just remove it.